eth/types/segment_test.go

```diff
@@ -35,9 +36,9 @@ func TestSegmentFlatten(t *testing.T) {
 	s := Segment{
 		StreamID:              "abcdef",
 		SegmentSequenceNumber: big.NewInt(1234),
-		DataHash:              ethcommon.RightPadBytes([]byte("browns"), 32),
+		DataHash:              ethcommon.BytesToHash(ethcommon.RightPadBytes([]byte("browns"), 32)),
 	}
-	if !bytes.Equal(ethcommon.Keccak256(s.Flatten()), s.Hash().Bytes()) {
+	if !bytes.Equal(crypto.Keccak256Hash(s.Flatten()).Bytes(), s.Hash().Bytes()) {
```

Comment:
We can use `crypto.Keccak256()` (it returns `[]byte`)

Reply:
I was thinking the same that they are not used anymore but wasn't sure. 

I'm in favor of removal then, or do you want to keep the fixed tests as reference just in case?

Reply:
I'm ok with removing

Reply:
I removed the files and their failing tests and rebased to drop the pervious commits.

Reply:
Should remove the test files as well?

Reply:
I did but must have been reset when rebasing I guess.